config-sample.yml

```diff
@@ -9,7 +9,6 @@ copperegg:
   - MS_NET_CLR
   - MS_ASP_NET
   - MS_Web_Services
-  - MS_MSSQL
```

Comment:
Why are we making changes to this sample file ? If changes are made in the way config works, lets reflect in sample file as well.